### ISSUE TITLE
PS scores not consistent between versions

### ISSUE THREAD

**Issue by lonelyjoeparker** [lonelyjoeparker/befi-bats-gui#16]
Some suggestion that the PS implementation changed somewhere 0.7.2 -> 0.10.1 due to generalisation to N states. In particular, the fitch state set pruning algorithm may be implemented differently.

Testing this:

Input | PS output 0.7.2 | PS output 0.10.1
----- | ------------- | -----------
example.trees, (PS = ?) | 18.5 | 25.733...
simulation100.trees, (PS = 4) | 4 | 4
fully_balanced_16tips_2states.trees, (PS = 8) | 8 | 8
fully_balanced_16tips_4states.trees, (PS = 12) | 12 | 12
fully_balanced_16tips_4states-partiallyAgg.trees, (PS = 9...?) | 9 | 10
fully_balanced_8tips_4states-partiallyAgg.trees, (PS = 5...?) | 5 | 5
fully_balanced_16tips_4states-moreAgg.trees (?) | 7 | 9

Basically seems to be overcounting, but in the newer(?) one. assert verbose and inspect states...

NB stast for AI and MC still consistent.
